### PR TITLE
fix(charts): reformat small area charts for legibility

### PR DESCRIPTION
**Pull request by vijaysharwar**
Trying a few things here to resolve various x/y axis and comparative chart
layout issues as described in #430:

- Made all area charts wider aspect ratio, to improve y-axis scaling with
  outlier states that have high number of tests.
- Switched to viewBox attribute for chart SVGs to allow responsive scaling
  to their containers
- On smaller screens < 960px, go to single column. > 960px go to two column
  format, looking for sweet spot of legibility and comparative analysis btwn
  states with high vs low number of tests
- Tweaked tick text font size, improve number of ticks consistency
- Adjusted the small charts legend for better responsive layout
- Updated grid line colors to #b2bbbf

### Small screen:
Screen Shot 2020-04-08 at 5 31 37 PM

### Large screen:
Screen Shot 2020-04-08 at 5 31 57 PM

### CODE REVIEW

File: src/pages/dashboard/_SmallMultiplesContainer.js [hunk 1]
```diff
@@ -105,11 +105,12 @@ const SmallMultiplesContainer = () => {
...
-    if (!data[1] && !data[0]) return
-    // eslint-disable-next-line consistent-return
-    return max(data[1].values, d => calculateTotal(d))
-  }, [data.length])
+
+  const maxStateTests = useMemo(() => {
+    return max(data, stateData => {
+      return max(stateData.values, d => calculateTotal(d))
+    })
+  })
```
Comment (vijaysharwar) [inline src/pages/dashboard/_SmallMultiplesContainer.js:113]: @jeremiak I revised this logic determining the max Y scaling. Unsure why code was using the first two only originally, but maybe I'm missing some context.

**Comment by julialedur** [comment]
pasting this here icymi. 
a few other things: 
- legend on mobile needs to be smaller 
- chart should still be a small multiples on mobile 

@vijaysharwar though this is a fairly exploratory viz, there are certain aspects that should be set by us, like number of columns. otherwise it just gets too confusing and impossible for us to get our point across. let's implement your toggle button for per capita numbers. that way we can show:
- how NY's raw numbers are incredibly high 
- the comparison between states and how normalizing for population makes a difference

they should be tiny tiny though as this is not supposed to be a detailed zoomed in graphic. like this: 

example

 it will be clearer once we add a tooltip.

**Comment by vijaysharwar** [comment]
@julialedur I can update the PR, it would look something like this:

state-charts

At that scale, the numbers are completely unreadable. And you can't see the amount of testing at all for the vast majority of states.

Maybe that can be improved by putting a larger chart on the individual state page (accessed by the "See all data from state" text links, though.

On that note, those "See all data from state" links are now overwhelming/repetitive. Should we make the text pithier and/or font size smaller, if we're making the charts this tiny?

**Comment by julialedur** [comment]
can we do 7 per row and normalize by population? that way the rest of the states won't be so low and we don't have a lost one at the end

**Comment by vijaysharwar** [comment]
I think normalizing by population should be a separate issue.

**Comment by vijaysharwar** [comment]
@julialedur I think 7 columns only works on larger screens. I've handled this responsively in https://github.com/COVID19Tracking/website/pull/449/commits/f2fcc08f8b7cf61e5ab55f2c9a3f77b480cfd143.

I also tried making the legend smaller, but if it still doesn't work please provide a mockup. Having trouble fitting that much info and keeping it legible/accessible.

### <480px

Screen Shot 2020-04-08 at 7 07 51 PM

### 480-799px

Screen Shot 2020-04-08 at 7 09 15 PM

### 800-1199px

Screen Shot 2020-04-08 at 7 10 38 PM

### 1200px and above

Screen Shot 2020-04-08 at 7 11 14 PM

**Comment by julialedur** [comment]
that looks nice! thanks for making it responsive too :) 

my thoughts: 
- keep only 100,000 and 200,000 ticks (it will remove noise) 
- we'll definitely need to normalize by pop. rn the chart doesn't tell much of a story (i'll create another issue for this) 
- let's change the last key in the legend to only "Stay-at-home order" and add a note below the chart that says "**Note:** Only statewide stay-at-home orders are included; Dates mark when the orders went into effect." a bit smaller than body text.

**Comment by vijaysharwar** [comment]
@julialedur if they're this small, the text and ticks are meaningless. Should we just remove them altogether? (We can keep them for the larger US totals chart at top of dashboard)

**Comment by vijaysharwar** [comment]
Here's where we're at with 17e3d49. I put the new "Note" above the legend, because otherwise it conflicts with the legend stickiness-on-scroll functionality.

Not sure what you mean with "make mobile version collapsible", but sounds like it should be a separate issue. With many contributors, best to keep PRs short-lived and concise to avoid merge  conflicts.

Screen Shot 2020-04-09 at 12 31 33 PM

**Comment by julialedur** [comment]
- sure, let's remove the tick labels and the grid. then we'll add a tooltip with this info.

- let's move the note to below the chart, it doesn't make sense above it. if you put it outside the chart's div it won't mess with the scroll functionality. 

- i'll open an issue to deal with the collapsible mobile version

**Comment by vijaysharwar** [comment]
@julialedur cool, how's this? 86f6e7c

Screen Shot 2020-04-09 at 2 38 05 PM

**Comment by julialedur** [comment]
oh sorry, i meant all the way at the bottom, after the chart! 
and Only should be capitalized ;)

**Comment by julialedur** [comment]
looks great otherwise! so much cleaner without grids and labels

**Comment by vijaysharwar** [comment]
Cool :)

What about doing it with an asterisk, like a footnote? That way it's linked to the legend, and people are less likely to miss it.

legend-note

**Comment by julialedur** [comment]
Great!

**Comment by vijaysharwar** [comment]
Done in 1b05594. I think this should be good to merge. cc @jeremiak @goleary @kevee

File: src/pages/dashboard/_SmallMultiplesContainer.js (hunk 1 above)
Comment (goleary) [reply src/pages/dashboard/_SmallMultiplesContainer.js]: we originally had NY going off the graph to demonstrate how bad things have gotten there.

**Pull request merged**